### DISCUSSION TITLE
How to make running program indicators thicker and easier to see?

### DISCUSSION THREAD

**Discussion by MinuteNewt** [linuxmint/discussions#415] [Questions] [answered]
Hello. I want to make the running program indicators thicker and easier to see. By the running program indicator, I don't mean the panel. I mean those very thin horizontal lines which are displayed under each running program, see screenshot. The programs that I have running are Firefox, Master PDF, Tauon Music Player and Ksnip. I want to make those blue indicators thicker, because although they are easy to see on the screenshot, I can't see them that well on my physical monitor (my eyesight isn't optimal). 

I know you can probably edit the theme's CSS file, but I don't know which bit of code to edit. The Desktop theme I'm using is Mint-Y-Dark-Aqua. Can someone tell me which line of CSS to edit? I asked about this in the Mint forum and one member tried to help me, but they couldn't find the line either. They said I should ask here on Github. They were able to tell me that the colour code is #1f9ede. See discussion [here](https://forums.linuxmint.com/viewtopic.php?p=2463544#p2463544).

![Open app indicators](https://github.com/linuxmint/discussions/assets/28846477/6e4d741d-fe22-4183-baaa-e6ec8659f212)

**Comment by anaximeno** [comment, marked as answer]
You'll have to change the file `/usr/share/themes/<theme-name>/cinnamon/cinnamon.css`, for the **Mint-Y-Dark-Aqua** it would be `/usr/share/themes/Mint-Y-Dark-Aqua/cinnamon/cinnamon.css` and modify the classes `.grouped-window-list-item-box.top, .grouped-window-list-item-box.bottom` and change the value for the `border-bottom-width`:

![screenshot-auto_1715258355](https://github.com/linuxmint/discussions/assets/63073056/923e6131-bded-44ca-9fb7-cc18a9fca514)

You may also want to change the `border-left-width` and `border-right-width` for the `grouped-window-list-item-box.left` and `grouped-window-list-item-box.right` respective classes.

After changing the values restart Cinnamon it the result should be applied to the window group items (in the screenshot you can see the result after changing the value to 5px).

**Reply by MinuteNewt** [reply]
Thank you, I modified `.grouped-window-list-item-box.top, .grouped-window-list-item-box.bottom` and increased the `border-bottom-width` from the default 2px to 4px which seems to be enough for me. Thank you so much for helping me, it's now much easier for me to see which programs are running when they are minimized!

**Reply by anaximeno** [reply]
Keep in mind that this may not be a permanent change, this change may be reset if the mint-y theme is updated by the system.

**Reply by MinuteNewt** [reply]
Yeah, I had a hunch that the changes could be reverted by OS version updates at the latest. I'll re-apply the changes as needed. Thanks again.

**Comment by raulrmar** [comment]
Thank you very much. I was also having the same difficulty.
